assets/_scss/themes/_usa.scss

```diff
@@ -16,13 +17,13 @@ li {
 }
 
 a {
-  color: $color-main;
+  color: $color-primary;
```

Comment:
This is good discussion around themes but we should move this to its own issue, outside of the typography PR.